Nos/Models/EditableNoteText.swift

```diff
@@ -16,34 +16,35 @@ struct EditableNoteText: Equatable {
         nsAttributedString.string
     }
     
-    private static var font = UIFont.preferredFont(forTextStyle: .body)
-    
     lazy var defaultAttributes: AttributeContainer = {
         AttributeContainer(defaultNSAttributes)
     }()
     
-    var defaultNSAttributes: [NSAttributedString.Key: Any] = [
-        .font: font,
-        .foregroundColor: UIColor.primaryTxt
-    ]
-    
+    var defaultNSAttributes: [NSAttributedString.Key: Any]
+
     var isEmpty: Bool {
         nsAttributedString.string.isEmpty
     }
     
     // MARK: - Init
     
     init() {
-        self.attributedString = AttributedString()
+        self.init(nsAttributedString: NSAttributedString(string: ""))
     }
     
     init(string: String) {
-        // We initialize this twice to avoid an error about `defaultAttributes` not being around before `self`.
-        self.attributedString = AttributedString(string)
-        self.attributedString = AttributedString(string, attributes: defaultAttributes)
+        self.init(nsAttributedString: NSAttributedString(string: string))
     }
     
-    init(nsAttributedString: NSAttributedString) {
+    init(
```

Comment:
UIColor.primaryTxt doesn't exist in unit tests, so this is a workaround. We enable the unit tests to instantiate EditableNoteText with available colors, like UIColor.black, while maintaining the default values for the rest of the app.